agent/agents/postgres/pgstatmonitor/pgstatmonitor_models.go

```diff
@@ -177,15 +177,15 @@ func NewPgStatMonitorStructs(v pgStatMonitorVersion) (*pgStatMonitor, reform.Vie
 			field{info: parse.FieldInfo{Name: "WalBytes", Type: "int64", Column: "wal_bytes"}, pointer: &s.WalBytes})
 	}
 
-	if v <= pgStatMonitorVersion10PG12 {
+	if vPG <= 12 {
 		fields = append(fields,
 			field{info: parse.FieldInfo{Name: "TotalTime", Type: "float64", Column: "total_time"}, pointer: &s.TotalTime},
 			field{info: parse.FieldInfo{Name: "MinTime", Type: "float64", Column: "min_time"}, pointer: &s.MinTime},
 			field{info: parse.FieldInfo{Name: "MaxTime", Type: "float64", Column: "max_time"}, pointer: &s.MaxTime},
 			field{info: parse.FieldInfo{Name: "MeanTime", Type: "float64", Column: "mean_time"}, pointer: &s.MeanTime},
 			field{info: parse.FieldInfo{Name: "StddevTime", Type: "float64", Column: "stddev_time"}, pointer: &s.StddevTime})
 	}
-	if v >= pgStatMonitorVersion10PG13 {
```

Comment:
This is actual fix, columns are not related to PGSM and PG version, but only to PG version. Everything else is small refactor of naming.